tutorials/raw/plot_annotating_raw.py

```diff
@@ -0,0 +1,225 @@
+# -*- coding: utf-8 -*-
+"""
+Annotating continuous data
+==========================
+
+This tutorial describes adding annotations to a Raw object, and how annotations
+are used in later stages of data processing.
+
+.. contents:: Page contents
+   :local:
+   :depth: 1
+
+As usual we'll start by importing the modules we need, loading some example
+data, and cropping it to save memory:
```

Comment:
and for illustration purpuses, we'll crop it to save memory

Reply:
This brings a new question. Why can't I do something like this (it seems more natural to me than cropping afterwards):
```py
sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
                                    'sample_audvis_raw.fif')
raw = mne.io.read_raw_fif(sample_data_raw_file, tmin=0, tmax=0.60, verbose=False)
```